portfolio/src/main/webapp/index.html

```diff
@@ -6,14 +6,27 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:lightsteelblue;">
```

Comment:
You can put this into a .css file if you want to reuse this styling elsewhere on your site.